mittab/apps/tab/outround_pairing_views.py

```diff
@@ -0,0 +1,639 @@
+import random
```

Comment:
General comments on this file: Lots of pretty long view functions. Can you extract some helpers? Even if it doesn't reduce duplication, just for readability

Ex: a `get_outround_options()` function rather than appending to that variable within the view function

Reply:
Added a `get_outround_options` function.  By way of how the view functions tend to be setup in `pairing_views.py` i'm not sure whether I should write it basically from scratch better, or leave it how it is now to be consistent.